cmd/user-auth/main.go

```diff
@@ -1,22 +1,22 @@
 package main
 
 import (
-	"fmt"
-	"os"
-
 	"github.com/alecthomas/kong"
 
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/users"
+	"github.com/edulinq/autograder/internal/cmd"
 	"github.com/edulinq/autograder/internal/config"
-	"github.com/edulinq/autograder/internal/db"
 	"github.com/edulinq/autograder/internal/log"
 	"github.com/edulinq/autograder/internal/util"
 )
 
 var args struct {
 	config.ConfigArgs
+	cmd.CommonOptions
 
-	Email string `help:"Email for the user." arg:"" required:""`
-	Token string `help:"Token for the user." arg:"" required:""`
+	Email    string `help:"Email for the user." arg:"" required:""`
+	Password string `help:"Password for the user." arg:"" required:""`
```

Comment:
Why the change?
"Tokens" is technically more correct:
https://github.com/edulinq/autograder-server/blob/main/internal/model/user_server.go#L36